cmd/ceremony/main.go

```diff
@@ -58,16 +58,18 @@ func (kgc keyGenConfig) validate() error {
 	return nil
 }
 
+type PKCS11KeyGenConfig struct {
```

Comment:
You should now be able to move the config validation out of each of the ceremony config types and just do a `validate` method on each of these structs.